src/vs/workbench/parts/extensions/node/extensionsWorkbenchService.ts

```diff
@@ -703,6 +709,11 @@ export class ExtensionsWorkbenchService implements IExtensionsWorkbenchService,
 				if (extensionIdentifier) {
 					this.checkAndEnableDisabledDependencies(extensionIdentifier);
 				}
+			}
+			// {{SQL CARBON EDIT}}
+			, (error) => {
```

Comment:
Why do we need changes here?

Reply:
added comments

Reply:
I would presume vscode handles if a vsix extension install fails somewhere? They really don't handle that?